### DISCUSSION TITLE
Tenders processing parties (BT-18)

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#452] [Q&A] [answered]
Hi labs !
Can you tell me please where is the cbc:EndpointID in <cac:TenderRecipientParty> ? 

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/a27d1488-f06f-40be-a062-1273a98b3f14)

Thanks and regards !
Renato

**Comment by YvesJo** [comment, marked as answer]
If you are referring to "Submission URL" (BT-18), you'll find the field inside the group "GR-Lot-SubmissionMethod" along with other Submission Eletronic related information (BT-17, BT-19, BT-745)
If you refer to the "Organisation eDelivery Gateway" (BT-509), you'll find it where the touchpoint you're referring to (i.e. TPO-0001) is defined
